app/models/season/phase_switcher.rb

```diff
@@ -4,6 +4,16 @@ def self.season
     Season.current
   end
 
+  def self.switcher(phase)
+    @phases = {
+        'Proposals' => :fake_proposals_phase,
```

Comment:
I'm thinking if we should drop the human readable mapping altogether and pass the method names around as strings instead. Reason: the magic values are human readable/understandable, but no one ever sees them. They are hidden in a select box and passed around as POST parameters.

If we decide to use the method names directly, it's imperative to whitelist the values that get passed to `send` / `public_send`. It would be a gaping security hole otherwise.